dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -15,7 +15,7 @@
     private String userDefinedTypeName;
     private Class<?> javaType;
     private int position; // zero-based index of parameter in procedure (-1 for ret value) or column in table
-    protected StatementExecution cs;
+    protected DbStatement cs;
```

Comment:
Should we change `cs` as it might suggest that the `DbStatement` wraps a `CallableStatement`?